x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plugin/EsqlFeatures.java

```diff
@@ -179,14 +179,9 @@ public class EsqlFeatures implements FeatureSpecification {
      */
     public static final NodeFeature RESOLVE_FIELDS_API = new NodeFeature("esql.resolve_fields_api");
 
-    /**
-     * Support categorize
-     */
-    public static final NodeFeature CATEGORIZE = new NodeFeature("esql.categorize");
```

Comment:
Oh yikes. I hadn't noticed the feature vs capability. You can't really remove these once they've left snapshot-land. But this one stayed in snapshot-land forever so I think it's safe to remove it.

Reply:
And I missed the pretty explicit:
```
Don't make more of those - add them to {@link EsqlCapabilities} instead.
```